README_PROCESS.md

```diff
@@ -0,0 +1,221 @@
+
+* Create a local branch to represent the cherry pick, for example
+  `git checkout -b cherry-pick-NNN`.
+
+* Run `git cherry-pick -x -m 1 <commit>` where `<commit>` is the merge commit of
+  some merged PR that we want to cherry pick from "main".
+  Resolve any merge conflicts if necessary.
+
+* Use `git commit --amend` to rewrite the commit message.
+  Rewrite the commit message from:
+
+  ```
+  Merge pull request #NNN from branch
+
+  Summary of PR
+
+  (cherry picked from commit ID)
+  ```
+
+  to:
+
+  ```
+  Summary of PR
+
+  Cherry pick #NNN from main
+  (cherry picked from commit ID)
+  ```
+
+* If the merge was not clean, verify that the specification still builds and
+  looks OK.
+
+* Push the branch to GitHub and create a new PR.
+  Be sure to change the PR "base" branch to "sycl-xxxx".
+
+* Once the CI tests pass, you can immediately merge the PR with a PR comment
+  like "Merging because the WG already approved #NNN".
+
+Although this process is a bit more verbose, it provides several advantages:
+
+* Each cherry picked commit references both the commit ID in the main branch and
+  the GitHub PR number, which allows for easy navigation in the git history.
+
+* By making changes to the "sycl-xxxx" branch via PRs, GitHub can automatically
+  create release notes for the next update revision of the specification.
+  If we had instead merged the commit manually, we would lose this ability.
+
+  **TODO:** It appears that the GitHub automatically-generated release notes do
+  not show the correct author for these cherry-picked PRs.
+  This author is displayed as the author of the person who creates the cherry
+  picked PR, which is normally the specification editor.
+  I tried setting the "author" and "committer" fields of the git commit before
+  it is pushed to the PR, but this doesn't help.
+  I think GitHub determines the author from the owner of the PR (not from any
+  field in the commit), and GitHub does not provide a way to change the owner of
+  a PR.
+
+After cherry picking PRs from "main" to "sycl-xxxx", it's useful to compare the
+two branches and check that there are no unexpected differences:
+
+```
+$ git diff sycl-2020 main
+```
+
+
+## Publishing an update revision
+
+**TODO:** These are my notes from publishing SYCL 2020 revision 9.
+We should check this process again when publishing revision 10 to make sure they
+are accurate.
+
+The working group normally accumulates a number "clarification" PRs in this
+GitHub repository before publishing them as an update revision of the current
+specification.
+When the working group decides it is time to publish an update revision, we
+follow this process.
+
+As noted above, the currently published major version may either correspond to
+the "main" branch or to the "sycl-xxxx" branch, depending on release cycle.
+The instructions below refer to the branch that contains the currently published
+major version as `<current>`.
```

Comment:
It's still not clear to me how to determine which branch to use here.

If we're publishing an update, why wouldn't it always be the "sycl-xxxx" branch?